CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ FILE(WRITE ${CMAKE_CURRENT_LIST_DIR}/docs/version
         )
 
 if(NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE "RelWithDebInfo" CACHE STRING
+    set(CMAKE_BUILD_TYPE "Release" CACHE STRING
```

Comment:
3. So, will a developer get `CMAKE_BUILD_TYPE=Release` by default? They have to set `CMAKE_BUILD_TYPE=RelWithDebInfo` manually?

Reply:
1. Please see https://github.com/taichi-dev/taichi/issues/1017#issuecomment-630836743
2. You still get the backtrace but some detailed debug symbols might be missing. I never use those symbols anyway
3. Yes.